standard/variables.md

```diff
@@ -12,7 +12,7 @@ As described in the following subclauses, variables are either ***initially assi
 
 ### 9.2.1 General
 
-C# defines seven categories of variables: static variables, instance variables, array elements, value parameters, reference parameters, output parameters, and local variables. The subclauses that follow describe each of these categories.
+C# defines eight categories of variables: static variables, instance variables, array elements, value parameters, input parameters, reference parameters, output parameters, and local variables. The subclauses that follow describe each of these categories.
```

Comment:
```suggestion
C# defines nine categories of variables: static variables, instance variables, array elements, value parameters, input parameters, reference parameters, output parameters, local variables, [and an almost fanatical devotion to the Pope](https://youtu.be/Cj8n4MfhjUc?t=47). The subclauses that follow describe each of these categories.
```

Reply:
(I'm trusting @BillWagner not to actually commit this...)

Reply:
I am so tempted....